views/incubator/demo/stainless.tpl

```diff
@@ -0,0 +1,28 @@
+<p>
+    The <a href="https://factory.c4dt.org/incubator/stainless/demo/">Stainless Demonstrator</a>
+    shows how Stainless helps develop Smart
+    Contracts free of errors. It provides an interactve environment
+    where the user can verify prewritten Smart Contracts, deploy them on
+    a blockchain, and call their functions.
+</p>
+<p>
+    A first introductory scenario involves a Candy Shop that maintains
+    its candy balance with a simple Smart Contract, which prevents people
+    from cheating by taking away more candies than are available. A second
+    scenario illustrates a bug in a Smart Contract that appeared on the
+    Ethereum blockchain a few years ago, and led to a controversial fork
+    to prevent significant monetary loss. It also explains how Stainless
+    could have prevented it.
+</p>
+
+<p>The following screenshot shows the start of the demonstrator. If you want
+    to discover for yourself, you can try to change the contract with (1). But a
+    better starting point is the tutorial, which you'll find in (2).</p>
```

Comment:
I would not suggest to the user that they try to "discover", but rather that they stick to the tutorial at first. It's already confusing enough as it is ;-)

Something like:
```suggestion
<p>The following screenshot shows the start of the demonstrator. The tutorial
    will guide you through the various elements of the interface and help you
    on your way.</p>
```